### ISSUE TITLE
Get-NixLog not working in v0.0.4

### ISSUE THREAD

**Issue by webmastir** [StartAutomating/PowerNix#11]
`Get-NixLog` does not seem to work in the v0.0.4.

```
PS /home/testuser> Get-InstalledModule PowerNix | select Version
0.0.4


PS /home/testuser> $PSVersionTable | select PSVersion
7.1.4


PS /home/testuser> Get-NixDistro
Ubuntu 20.04.3 LTS Ubuntu ubuntu debian  20.04.3 LTS (Focal Fossa)


PS /home/testuser> Get-NixLog -LogFilePath '/var/log/syslog'    
Write-Error: /home/testuser/.local/share/powershell/Modules/PowerNix/0.0.4/Get-NixLog.ps1:168
Line |
 168 |              & { process {
     |              ~~~~~~~~~~~~~
     | Unable to parse log
```

Snippet from /var/log/syslog:
```
Aug 24 00:00:01 linux-test systemd[1]: logrotate.service: Succeeded.
Aug 24 00:00:01 linux-test systemd[1]: Finished Rotate log files.
Aug 24 00:00:01 linux-test systemd[1]: man-db.service: Succeeded.
Aug 24 00:00:01 linux-test systemd[1]: Finished Daily man-db regeneration.
Aug 24 00:00:05 linux-test multipathd[692]: sda: add missing path
Aug 24 00:00:05 linux-test multipathd[692]: sda: failed to get udev uid: Invalid argument
Aug 24 00:00:05 linux-test multipathd[692]: sda: failed to get sysfs uid: Invalid argument
Aug 24 00:00:05 linux-test multipathd[692]: sda: failed to get sgio uid: No such file or directory
Aug 24 00:00:10 linux-test multipathd[692]: sda: add missing path
Aug 24 00:00:10 linux-test multipathd[692]: sda: failed to get udev uid: Invalid argument
```

**Comment by StartAutomating** [comment]
@joshcorr Want to take a look?

**Comment by joshcorr** [comment]
I'll take a look

**Comment by joshcorr** [comment]
Looks like the regex does not like the `-` in the computer name.

**Comment by joshcorr** [comment]
@webmastir Could you please try version `0.0.5` from the PowerShell Gallery? This should fix your specific issue.

**Comment by webmastir** [comment]
@joshcorr No dice. It throws this MethodInvocationException error for each line in the file:

```
PS /home/testuser> Get-NixLog -LogFilePath '/var/log/syslog'

MethodInvocationException: /home/testuser/.local/share/powershell/Modules/PowerNix/0.0.5/Get-NixLog.ps1:150
Line |
 150 |                          [PSCustomObject][ordered]@{
     |                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Exception calling "ParseExact" with "3" argument(s): "String 'Sep  7 00:00:37' was not recognized as a valid DateTime."

MethodInvocationException: /home/testuser/.local/share/powershell/Modules/PowerNix/0.0.5/Get-NixLog.ps1:150
Line |
 150 |                          [PSCustomObject][ordered]@{
     |                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Exception calling "ParseExact" with "3" argument(s): "String 'Sep  7 00:00:37' was not recognized as a valid DateTime."

MethodInvocationException: /home/testuser/.local/share/powershell/Modules/PowerNix/0.0.5/Get-NixLog.ps1:150
Line |
 150 |                          [PSCustomObject][ordered]@{
     |                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Exception calling "ParseExact" with "3" argument(s): "String 'Sep  7 00:00:37' was not recognized as a valid DateTime."
```

Looks like you saw the same thing in https://github.com/StartAutomating/PowerNix/issues/14#issue-985917805

**Comment by webmastir** [comment]
Looks like you saw the same thing in https://github.com/StartAutomating/PowerNix/issues/14#issue-985917805

**Comment by joshcorr** [comment]
The method we are using for parsing the string Timestamp in the file requires a space and two days and can't handle the day with two spaces and one digit. Hopefully fixed in #15

**Comment by joshcorr** [comment]
@webmastir version 0.0.6 published to the gallery should work now. Sorry for the bugs. Please feel free to open another issue if you find something else.
